src/codegeneration/FromOptionsTransformer.js

```diff
@@ -113,9 +114,8 @@ export class FromOptionsTransformer extends MultiTransformer {
     if (options.typeAssertions) {
       // Transforming member variables to getters/setters only make
       // sense when the type assertions are enabled.
-      if (options.typeAssertions) {
-        append(TypedMemberVariableTransformer);
-      }
+      append(TypeToExpressionTransformer);
```

Comment:
The comment still applies but it needs to go with the `TypedMemberVariableTransformer`.